mordred/task_panels.py

```diff
@@ -171,19 +164,28 @@ def __configure_kibiter(self):
 
         if 'panels' not in self.conf:
             logger.warning("Panels config not availble. Not configuring Kibiter.")
-            return
+            return False
 
         kibiter_major = es_version(self.conf['es_enrichment']['url'])
         if kibiter_major == "6":
-            # Force the creation of the .kibana index
-            res = requests.post(ES6_KIBANA_INIT_URL, headers=ES6_KIBANA_INIT_HEADERS,
-                                data=ES6_KIBANA_INIT_DATA)
-            try:
-                res.raise_for_status()
-            except Exception as ex:
-                print(ES6_KIBANA_INIT_HEADERS)
-                print(ES6_KIBANA_INIT_DATA)
-                logger.error("Can not create the .kibana in ES6 %s", ex)
+            if self.conf['panels']["kibiter_url"] and self.conf['panels']["kibiter_version"]:
+                # Force the creation of the .kibana index
+                # We don't have this data so it just works for this value
+                k6_init_url = self.conf['panels']["kibiter_url"]
+                k6_init_url += "/api/kibana/settings/indexPattern:placeholder"
+                k6_init_data = '{"value": "*"}'
+                k6_init_headers = {
+                    "Accept": "application/json",
+                    "Content-Type": "application/json",
+                    "kbn-version": self.conf['panels']['kibiter_version']
+                }
+
+                res = requests.post(k6_init_url, headers=k6_init_headers,
```

Comment:
If you extend the exception handle to cover this call to requests.post, it will catch also the case when the "name or service not found" error happens. Do you think that's worthwhile?

Reply:
Yes, good point. My fault not to include the POST call in the try. This is why it was failing to you. Changed!